sgf-viewer/src/components/Section.svelte

```diff
@@ -6,42 +6,40 @@
     let sgfPath: string;
 </script>
 
-<div class="centerflex" id={section["dir_name"]}>
```

Comment:
this entire div is no longer needed — class "centerflex" isn't used.

besides moving the id to the `<h3>`, everything else here is an indent change